### PR TITLE
Migration to AndroidX #324

### PR DESCRIPTION
**Pull request by fegol**
Migration ribs to AndroidX

### CODE REVIEW

**Comment by fegol** [comment]
Build failed because i updated android target sdk to 28 version

File: android/gradle/dependencies.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 def versions = [
     dagger: "2.22.1",
-    support: '25.1.1',
+    support: '1.0.0',
```
Comment (ShaishavGandhi) [inline android/gradle/dependencies.gradle]: Can you separate out the versions? The main advantage with androidx is that artifacts can have their own versions.

Comment (fegol) [reply android/gradle/dependencies.gradle]: done

File: android/gradle/dependencies.gradle [hunk 2]
```diff
@@ -64,9 +64,9 @@ def build = [
 ]
 
 def support = [
```
[the remark this answers is not in the document]
Comment (fegol) [reply android/gradle/dependencies.gradle]: done

**Comment by fegol** [comment]
Fixed

File: android/gradle/dependencies.gradle [hunk 3]
```diff
@@ -16,7 +16,11 @@
 
 def versions = [
     dagger: "2.22.1",
-    support: '25.1.1',
+    androidx: [
+            annotations: '1.1.0',
```
Comment (ShaishavGandhi) [inline android/gradle/dependencies.gradle]: Nit: Can you make this 4 spaces instead?

Comment (fegol) [reply android/gradle/dependencies.gradle]: done

**Pull request merged**